src/test/scala/com/databricks/spark/avro/AvroSuite.scala

```diff
@@ -133,7 +147,7 @@ class AvroSuite extends FunSuite {
         StructField("bool_array", ArrayType(BooleanType), true),
         StructField("long_array", ArrayType(LongType), true),
         StructField("double_array", ArrayType(DoubleType), true),
-        StructField("decimal_array", ArrayType(DecimalType(5, 5)), true),
+        StructField("decimal_array", ArrayType(DecimalType(10, 0)), true),
```

Comment:
/cc @davies to look at decimal changes here. Had to change things here to avoid nulls being output by conversion.